### ISSUE TITLE
SAML SSO in NewDot - iOS and Android UI

### ISSUE THREAD

**Issue by sakluger** [Expensify/App#28217]
Tracking GH issue: https://github.com/Expensify/Expensify/issues/294583
Relevant design doc section: https://docs.google.com/document/d/1xa68adybahTxucjSqvPJ2MZCZvaBoP1_vtYk5hTikGU/edit#heading=h.72o1331fkwa2

**Comment by NikkiWines** [comment]
Focusing on web / mweb first, then on platform specific logic

**Comment by sakluger** [comment]
No updates, @NikkiWines what's the ETA on this one? Would it still make sense to ask Infinite Red to help with this one?

**Comment by NikkiWines** [comment]
Similar to the desktop update - we're still looking at getting web/mweb over the line so it would be good to get external/agency eyes on the platform specific logic

**Comment by sakluger** [comment]
Not overdue, we're just working on other platforms first.

**Comment by NikkiWines** [comment]
See update [here](https://github.com/Expensify/Expensify/issues/294583#issuecomment-1754774720)

**Comment by sakluger** [comment]
@NikkiWines you said that you started working on this one?

**Comment by NikkiWines** [comment]
Yep, should be handled in https://github.com/Expensify/App/pull/29526 ^^

**Comment by NikkiWines** [comment]
iOS/Android PR is in the works!

**Comment by sakluger** [comment]
Not overdue, @NikkiWines is actively working on the PR.

**Comment by sakluger** [comment]
It looks like @NikkiWines is making progress on the PR, I see new commits from two hours ago.

**Comment by NikkiWines** [comment]
UI is looking good and the backend PR is in review. Hoping to get https://github.com/Expensify/App/pull/30027 updated + merged into my branch so we can be all set to merge https://github.com/Expensify/App/pull/29526 once the backend changes are live.

**Comment by NikkiWines** [comment]
Still waiting on https://github.com/Expensify/Web-Expensify/pull/39366 to be reviewed

**Comment by NikkiWines** [comment]
https://github.com/Expensify/Web-Expensify/pull/39366 is merged, hoping to get https://github.com/Expensify/App/pull/29526 reviewed and merged next week!

**Comment by NikkiWines** [comment]
Created an issue to handle payment for PR review: https://github.com/Expensify/App/issues/32701

**Comment by NikkiWines** [comment]
Oops this can be closed now!
